### PR TITLE
i360 integrations

### PR DESCRIPTION
**Pull request by xiaojun0901**

### CODE REVIEW

File: source/includes/_integrations.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
+# Integrations
+
+## I360 Integration
+
+ArcSite offers a comprehensive integration solution for I360, designed to enhance the experience of users who rely on both ArcSite and I360. This integration provides a range of extended functionalities, empowering your projects to:
+
+- Associate `I360 Appointments` with ArcSite `Projects`
+- Automatically synchronize `Drawing PDF data (including PDF files of Drawings and Proposal PDFs)` for associated Projects
+- Proactively transmit `Product Lines` information to I360 Appointments for associated Projects, accessible via both the App and User Site
```
Comment (ryanwang520) [inline source/includes/_integrations.md]: Proactively  意思不对，此主动非彼主动。Proactively transmit -> Push, proactive出现的地方都要去掉

File: source/includes/_integrations.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+- Enable subscription to the [Proposal Exported in App associated with i360](#proposal-exported-in-app-associated-with-i360) webhook
```
Comment (ryanwang520) [inline source/includes/_integrations.md]: Proposal Exported in App associated with i360 -> Proposal Exported in App

File: source/includes/_integrations.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+3. On the ArcSite website, select the uploaded Drawing.
+4. In the Takeoff & Estimate Tab, ensure the 'Send to I360' button is visible.
+   ![Untitled](https://cdn-public.arcsiteapp.com/api_docs_images/send_to_i360_png.png)
+5. If the 'Send to I360' button appears, it confirms the successful association, allowing the Project to utilize data synchronization.
+
+<aside class='notice'>
+This concludes the validation process for the integration. If you encounter any issues during validation, please review the steps and ensure that the Project was created successfully using the <a href="#create-project">Create Project</a> API.
+</aside>
+
+### Connecting Products between ArcSite and I360
```
Comment (ryanwang520) [inline source/includes/_integrations.md:45]: 可以把网站对应页面的url放到这里，只有一张截图也不清楚到哪里看, 这里应该再给一个没有connect过的截图吧

File: source/includes/_integrations.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+**Automatic PDF Data Synchronization**
+
+- **Synchronization Timing**:
+  ![Untitled](https://cdn-public.arcsiteapp.com/api_docs_images/upload_to_cloud_png.png)
+  ![Untitled](https://cdn-public.arcsiteapp.com/api_docs_images/export_proposal_png.png)
+  Whenever user Drawing information changes and these alterations sync to the cloud, ArcSite initiates an automatic data synchronization task to update the relevant I360 Appointment as necessary.
+
+- **Synchronization Logic**:
+  The decision to initiate a push to I360 is based on the configuration in the **`is_push_drawing`** and **`is_push_proposal`** fields when associating the project. When a push request is made, a task generates the latest Drawing PDF data and exports all PDFs of the user's Proposal Templates using the latest Drawing data. These files are then automatically pushed to I360, overwriting any existing attachments.
```
Comment (ryanwang520) [inline source/includes/_integrations.md]: is_push_drawing 和 is_push_proposal 这两个变量名可以改一下， sync_drawing_pdf, sync_proposal_pdf

File: source/includes/_integrations.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+  Users can locate these files in the Attachments section of the Appointment associated with the Project in I360.
+
+### Drawing Line Items and Pricing Information
+
+The synchronization of Product data is manual.
+
+- **Synchronization Timing**:
+  - Within the ArcSite app, after a successful Export Customer Proposal, viewing the generated PDF and selecting "Done" in the top left corner triggers a pop-up asking "Did you sell this project?" If the user answers "Yes," ArcSite initiates an asynchronous task to push the Drawing Line Item data to I360.
+    ![Untitled](https://cdn-public.arcsiteapp.com/api_docs_images/did_you_sell_project_png.png)
+  - On the User Site, users can click "Send to I360" in the Drawing's Takeoff & Estimate Tab to initiate the Product data push to I360.
```
Comment (ryanwang520) [inline source/includes/_integrations.md]: On the User Site, users can click "Send to I360" in the Drawing's Takeoff & Estimate Tab to initiate the Product data push to I360.

->

You can also trigger a manual sync on the drawing detail page of the user site by clicking *Send to I360* in the Drawing's Takeoff & Estimate Tab.

File: source/includes/_integrations.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+### APIs for I360
+
+### Connected I360 Product
+
+In ArcSite, after setting up the I360 environment, you can connect an I360 Product to an ArcSite Product within the ArcSite website.
+
+To retrieve the connected I360 Product ID using an ArcSite Product ID, you can utilize this API:
+
+```shell
+curl 'https://api.arcsite.com/v1/i360/connected_product/<arcsite_product_id>/' \
```
Comment (ryanwang520) [inline source/includes/_integrations.md]: open ai 统一不加 trailing slash

'https://api.arcsite.com/v1/i360/connected_product/<arcsite_product_id>'

File: source/includes/_integrations.md (hunk 2 above)
Comment (xiaojun0901) [reply source/includes/_integrations.md]: 不区分I360了吗？如果用这个的话 到时候在 webhook 选择哪里 Salesforce， I360 会没办法区分

Comment (ryanwang520) [reply source/includes/_integrations.md]: 通过超链接的地址就可以区分了

**Pull request merged**